v3/src/components/axis/hooks/use-axis-bounds.ts

```diff
@@ -1,14 +1,15 @@
 import {useEffect, useState} from "react"
 import {AxisPlace} from "../axis-types"
 import {useGraphLayoutContext} from "../../graph/models/graph-layout"
+import {useAxisLayoutContext} from "../models/axis-layout-context"
```

Comment:
The `useGraphLayoutContext` hook should not longer be required, so its import can be removed.

Reply:
Actually it was still, wrongly, being used in useAxisBounds. So I changed useAxisBounds to refer to usAxisLayoutContext instead of useGraphLayoutContext.